docs/components/form.md

```diff
@@ -375,6 +375,66 @@ You can access those values from the `request.url`
 
 - [useSubmit][usesubmit]
 
+## Registration Form
+
+We can use Mutation Submissions in registration workflow.
+
+```tsx
+function RegisterForm() {
+  const actionData = useActionData();
+
+  return (
+    <Form method="post">
+      <label>Name</label>
+      <input type="text" name="name" />
+
+      <label>Email</label>
+      <input type="email" name="email" />
+
+      <label>Password</label>
+      <input type="password" name="password" />
+
+      <button type="submit">Register</button>
+
+      {actionData && actionData.error && (
```

Comment:
`actionData && actionData.error` can be simplified with conditional chaining + we should also use ternaries instead of && in JSX (see [Kent C. Dodds' "Use ternaries rather than && in JSX" blogpost](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) for the reasoning behind this)

```suggestion
      {actionData?.error ? (
```